air/src/stack/op_flags/mod.rs

```diff
@@ -676,10 +676,10 @@ pub fn get_op_bits(opcode: usize) -> [Felt; NUM_OP_BITS] {
     // initialise the bit array with 0.
     let mut bit_array = [ZERO; NUM_OP_BITS];
 
-    for i in 0..NUM_OP_BITS {
+    for i in bit_array.iter_mut().take(NUM_OP_BITS) {
```

Comment:
Why do we need `.take()` here? We are iterating over all elements of the array, right?

Reply:
I've removed it in the latest PR. I was using it based on clippy suggestion.